bundles/automation/org.eclipse.smarthome.automation.commands/src/main/java/org/eclipse/smarthome/automation/internal/commands/CommandlineTemplateProvider.java

```diff
@@ -103,11 +103,14 @@ public void exportTemplates(String parserType, Set<RuleTemplate> set, File file)
      * @see AutomationCommandsPluggable#importTemplates(String, URL)
      */
     public Set<RuleTemplate> importTemplates(String parserType, URL url) throws IOException, ParsingException {
-        InputStreamReader inputStreamReader = null;
         Parser<RuleTemplate> parser = parsers.get(parserType);
-        if (parser == null) {
-            inputStreamReader = new InputStreamReader(new BufferedInputStream(url.openStream()));
-            return importData(url, parser, inputStreamReader);
+        if (parser != null) {
+            InputStreamReader inputStreamReader = new InputStreamReader(new BufferedInputStream(url.openStream()));
```

Comment:
Well, the EnOcean base driver is a subproject on its own (and not included anywhere in the "normal" ESH build - it says that it runs on Java 1.4 here: https://github.com/eclipse/smarthome/tree/master/protocols/enocean

For the automation module, we more or less agreed not to bother developers with this, but rather have @marinmitev check the compatibility and create follow-up PRs if necessary.